src/editor/Editor.js

```diff
@@ -30,12 +31,15 @@ class Editor extends Component {
                     markdownValue={this.state.value} 
                     setEditorState={this.setEditorState} 
                     htmlValue={this.state.html}
-                    className={'toolbar'}/>
+                    className={'toolbar'}
+                    store={store}
```

Comment:
warum wird hier der store importiert und in zeile 42 durch die props reingereicht?